wiki/communities/community-hub.md

```diff
@@ -25,6 +25,30 @@ for those wanting to test their acc abilities against difficult requirements.
 
 ---
 
+## BeastSaber
```

Comment:
looks incomplete / doesn't make sense in terms of BeastSaber scope, mix of what BSMG does.

For clarification BSMG does not operate bsaber, thats a separate team. This may be a good reference to build a new description https://bsaber.com/beastsaber-rising-rebuilding-a-community-cornerstone/

Reply:
ah. this section was mostly pulled from their patreon. must be outdated. will fix

Reply:
waiting for a writeup from the beastsaber team. Most info available on their website is outdated